tabpy/tabpy_server/handlers/util.py

```diff
@@ -2,13 +2,17 @@
 from hashlib import pbkdf2_hmac
 from enum import Enum, auto
 
-
 class AuthErrorStates(Enum):
     NONE = auto()
     NotAuthorized = auto()
     NotRequired = auto()
 
 
+class AuthErrorStates(Enum):
```

Comment:
Why this class is duplicated?

Reply:
Oops, I mixed up my branches.  Will get that fixed right away